utils/_features.py

```diff
@@ -2047,6 +2047,16 @@ def iast_sink_reflection_injection(test_object):
         pytest.mark.features(feature_id=279)(test_object)
         return test_object
 
+    @staticmethod
+    def f_interoperability(test_object):
```

Comment:
should this name reflect that this is `f_otel_interoperability` ?

Reply:
Deleted the previous feature and [created a new one](https://dd-feature-parity.azurewebsites.net/Manage/Features/Edit?id=289) to use the `f_otel_interoperability` name :-)